playbook/app/pb_kits/playbook/pb_background/background.rb

```diff
@@ -16,14 +16,27 @@ class Background < Playbook::KitBase
                         values: ["blur", "fade", "scale", nil],
                         default: nil
 
+      prop :custom_color, type: Playbook::Props::String,
+                          default: nil
+
       def classname
         generate_classname("pb_background_kit", image_classname, separator: " ")
       end
 
+      def custom_background_color
+        "background-color: #{custom_color};"
+      end
+
     private
 
       def image_classname
-        image_url.present? ? "lazyload #{transition}" : "pb_background_color_#{background_color}"
+        if image_url.present?
+          "lazyload #{transition}"
+        elsif custom_color.present?
+          "pb_background_custom_color"
+        else
+          "pb_background_color_#{background_color}"
+        end
```

Comment:
When custom_color is present I think you want a value to be set dynamically : `"pb_background_#{custom_color}"`.

Reply:
I was thinking about that but when custom color is being used, that image classname really has no effect on the style. I could do it like that just so people could inspect the page and see the value. I just didn't want to leave it blank because then the classname has a blank space at the end.